app/src/main/java/com/capstone/weatherapp/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package com.capstone.weatherapp
 
 import android.os.Bundle
-import androidx.activity.ComponentActivity
+import android.view.View
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
 
-class MainActivity : ComponentActivity() {
+class MainActivity : AppCompatActivity() {
+    lateinit var cityList : ArrayList<City>
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.fragment_city_list)
+        val rvCities = findViewById<View>(R.id.rvCities) as RecyclerView
+
+        cityList = ArrayList<City>(List(100) { City("List Item", 72) })
+
+        val adapter = CityListAdapter(cityList)
```

Comment:
Let's move all of this adapter/recyclerView logic out of the MainActivity and into the Fragment you created. For now, all MainActivity should be doing is inflating the layout